### ISSUE TITLE
What is the age of the result?

### ISSUE THREAD

**Issue by Angus1996** [yoyohonyang/LearingFaceAgeProgression#2]
In the paper，different ages are listed out, like 31-40,41-50.
So, I want to know, how to get the other result?
Or，can I change the age of result image?

**Comment by voa18105** [comment]
Yes, I wonder for this also... how to change output age? I cannot find anything corresponding in the code
